templates/control/control.html

```diff
@@ -18,6 +18,7 @@
     <p><a href="/control/editemailpassword">Password and Email</a></p>
     <p><a href="/manage/folders">Submission Folders</a></p>
     <p><a href="/manage/tagfilters">Search Tag Filters</a></p>
+    <p><a href="/control/editsearchtagblacklist">Edit Community Tagging Blacklist</a></p>
```

Comment:
Rename the route to `/control/tagblacklist` and change the text to “Community Tagging Blacklist” (most of the other items in this list don’t include the action – we can make them all consistent later).